src/ontology/mondo-edit.obo

```diff
@@ -440510,6 +440508,7 @@ xref: SCTID:715655000 {source="MONDO:equivalentTo"}
 xref: UMLS:C4275067 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:907865"}
 is_a: MONDO:0007100 {source="Orphanet:85451"} ! familial amyloid neuropathy
 is_a: MONDO:0016340 {source="Orphanet:85451", source="https://orcid.org/0000-0001-5208-3432"} ! familial restrictive cardiomyopathy
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/12405 {source="https://orcid.org/0000-0002-0587-4693"} ! TTR
```

Comment:
the source for this relation should be the orphanet xref since this is where we found the information about the variation.

Reply:
Comment addressed